src/lib/jar.js

```diff
@@ -73,6 +73,21 @@ var Jar = {
         var expires = '; expires=' + new Date(new Date().setYear(new Date().getFullYear() + 1)).toUTCString();
         var path = '; path=/';
         document.cookie = obj + expires + path;
+    },
+    getSignedValue: function (cookieName, signedValue, callback) {
```

Comment:
hmm. yeah I see that. I was going with the name based on the fact that it is unsigning as part of its process, so as to suggest that it's getting what wan an originally signed value, but I see how that might be confusing...